packages/core/src/components/slider/multiSlider.tsx

```diff
@@ -419,6 +423,19 @@ export class MultiSlider extends AbstractPureComponent2<IMultiSliderProps, ISlid
         });
     };
 
+    private getLabelValues() {
+        const { labelStepSize, labelValues, min, max } = this.props;
+        let values: number[] = [];
+        if (labelValues) values = labelValues;
+        else {
```

Comment:
🤦 I really need to re-enable the lint rule for this... but please ensure that all if/else/for/while/etc... clauses are blocks enclosed by curlys

```suggestion
        if (labelValues !== undefined) {
            values = labelValues;
        } else {
```